Components/Submenus/Submenus.css

```diff
@@ -0,0 +1,61 @@
+.navigation-area {
+  background: hsl(var(--clr-white));
+  height: fit-content;
+  width: 70%;
+  padding: 1rem;
+  align-items: center;
+}
+
+.fa-search {
+  align-self: center;
+  margin: -0.25rem 0 0 0.25rem;
+}
+
+.search {
+  border: 3px solid hsl(var(--clr-dark));
+}
+
+.submenu {
+  margin-left: auto;
+}
+
+.fa-caret-down {
+  align-self: center;
+  margin-left: 0.25rem;
+}
+
+.dropdown-content {
+  padding: 0;
+  margin: 0;
+  text-align: left;
+  display: none;
+  background: hsl(var(--clr-white));
+  top: 100%;
+}
+
+.submenu-item .fa-shopping-cart,
+.fa-heart-pulse {
+  margin-left: 0.25rem;
+}
+
+.submenu-item > a {
+  padding: 0.5rem 0.68rem 0.5rem 0.25rem;
+  margin: 1rem 0.5rem 1rem 0.25rem;
```

Comment:
Use px for smaller values?

Reply:
https://discord.com/channels/914867137863118919/927786435334004787/940298016320028773